src/interactions/pointer.js

```diff
@@ -0,0 +1,159 @@
+      // closest point across facets to be visible.
+      const faceted = index.fi != null;
+      const facetState = faceted ? (state.facetState ??= new Map()) : null;
+
+      // The order of precedence when determining the point position is: px &
+      // py; the middle of x1 & y1 and x2 & y2; or lastly x & y. If any
+      // dimension is unspecified, we fallback to the frame anchor.
+      const {x: X, y: Y, x1: X1, y1: Y1, x2: X2, y2: Y2, px: PX, py: PY} = values;
+      const [cx, cy] = applyFrameAnchor(this, dimensions);
+      const px = PX ? (i) => PX[i] : X2 ? (i) => (X1[i] + X2[i]) / 2 : X ? (i) => X[i] : () => cx;
+      const py = PY ? (i) => PY[i] : Y2 ? (i) => (Y1[i] + Y2[i]) / 2 : Y ? (i) => Y[i] : () => cy;
+
+      let i; // currently focused index
+      let g; // currently rendered mark
+
+      function render(ii, ri) {
+        // When faceting, if more than one pointer would be visible, only show
+        // this one if it is the closest. This is a simple linear scan because
+        // we don’t expect many facets with simultaneously-visible pointers.
+        if (faceted) {
+          if (ii == null) {
+            facetState.delete(index.fi);
+          } else {
+            facetState.set(index.fi, ri);
+            if (facetState.size > 1) {
+              for (const [fi, r] of facetState) {
+                if (fi !== index.fi && r < ri) {
+                  ii = null;
+                  break;
+                }
+              }
+            }
+          }
+        }
+        if (i === ii) return; // the tooltip hasn’t moved
+        i = ii;
+        const I = i == null ? [] : [i];
+        if (faceted) (I.fx = index.fx), (I.fy = index.fy), (I.fi = index.fi);
+        const r = mark.render(I, scales, values, dimensions, context);
+        if (g) {
+          // When faceting, preserve swapped mark and facet transforms; also
+          // remove ARIA attributes since these are promoted to the parent. This
+          // is perhaps brittle in that it depends on how Plot renders facets,
+          // but it produces a cleaner and more accessible SVG structure.
+          if (faceted) {
+            const p = g.parentNode;
+            const ft = g.getAttribute("transform");
+            const mt = r.getAttribute("transform");
+            ft ? r.setAttribute("transform", ft) : r.removeAttribute("transform");
+            mt ? p.setAttribute("transform", mt) : p.removeAttribute("transform");
+            r.removeAttribute("aria-label");
+            r.removeAttribute("aria-description");
+            r.removeAttribute("aria-hidden");
+          }
+          g.replaceWith(r);
+        }
+        state.roots[renderIndex] = r;
+        return (g = r);
+      }
+
+      function pointermove(event) {
+        if (state.sticky || (event.pointerType === "mouse" && event.buttons === 1)) return; // dragging
+        let [xp, yp] = pointof(event);
+        if (faceted) (xp -= tx), (yp -= ty);
```

Comment:
```suggestion
        xp -= tx;
        yp -= ty;
```

Apply tx, ty translation even in the non-faceted case (the tipCell test)

Reply:
Yes, I fixed this and then broke it when staging changes. It’s fixed again.